TEMPLATE.md

```diff
@@ -0,0 +1,652 @@
+## Shortcuts from raulfraile, applied on Phpstorm by Pouzor ##
+
+All shortcuts start with the `sf` prefix and are both short and intuitive:
+
+### Controller ###
+
+`sfcontroller`
+
+``` php
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    public function indexAction()
+    {
+        return $this->render('.html.twig');
+    }
+}
+```
+
+`sfcontrollera`
+
+``` php
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    /**
+     * @Route("/", name="")
+     * @Template()
+     */
+    public function indexAction()
+    {
+
+    }
+}
+```
+
+`sfaction`
+
+``` php
+public function indexAction()
+{
+    return $this->render('.html.twig');
+}
+```
+
+`sfactiona`
+
+``` php
+/**
+ * @Route("/", name="")
+ * @Template()
+ */
+public function indexAction()
+{
+
+}
+```
+
+`sfem`
+
+``` php
+$em = $this->getDoctrine()->getManager();
+```
+
+`sfrepo`
+
+``` php
+$em->getRepository('Bundle:Repository');
+```
+
+`sfforward`
+
+``` php
+$this->forward('Bundle:Controller:action', array(), array());
+```
```

Comment:
We should remove this - forwarding is basically worthless, we practically removed it from the documentation :)